bans-core/src/main/java/space/arim/libertybans/core/config/Formatter.java

```diff
@@ -215,12 +229,16 @@ String getVariable() {
 		simpleReplacements.put(SimpleReplaceable.DURATION, durationFormatted);
 		simpleReplacements.put(SimpleReplaceable.START_DATE, formatAbsoluteDate(punishment.getStartDate()));
 		simpleReplacements.put(SimpleReplaceable.TIME_PASSED, formatRelative(timePassed));
+		simpleReplacements.put(SimpleReplaceable.TIME_PASSED_SIMPLE, formatRelativeSimple(timePassed));
 		simpleReplacements.put(SimpleReplaceable.END_DATE, formatAbsoluteDate(punishment.getEndDate()));
 		simpleReplacements.put(SimpleReplaceable.TIME_REMAINING, relativeEndFormatted);
+		simpleReplacements.put(SimpleReplaceable.TIME_REMAINING_SIMPLE, relativeEndFormattedSimple);
+		MessagesConfig.Formatting.PunishmentActiveDisplay display = messages().formatting().punishmentActiveDisplay();
+		simpleReplacements.put(SimpleReplaceable.IS_ACTIVE, (display != null) ? ((isActive) ? display.active() : display.expired()) : "Unknown");
```

Comment:
The null check isn't necessary, because the configuration will update itself by adding default configuration options if they do not exist.
```suggestion
		simpleReplacements.put(SimpleReplaceable.IS_ACTIVE, (isActive) ? display.active() : display.expired());
```

Reply:
That didn't work for me though, the tests kept failing because `punishmentActiveDisplay()` returned null. It worked flawlessly ingame.